openedx/core/djangoapps/schedules/models.py

```diff
@@ -24,10 +24,7 @@ class Schedule(TimeStampedModel):
         help_text=_('Date this schedule went into effect')
     )
     start_date = models.DateTimeField(
-        # TODO remove blank=True and null=True once data migration is complete in column rename release.
-        blank=True,
         db_index=True,
-        null=True,
```

Comment:
This line should have been the clue to tip us off that an AlterField would be run against this table.

Reply:
Yes unfortunately, it is.